### ISSUE TITLE
Text in FontDescriptor description no longer correct

### ISSUE THREAD

**Issue by lrosenthol** [pdf-association/pdf-issues#47]
In 9.6.2.1, Table 109, the entry for **FontDescriptor** includes the text:
>For the standard 14 fonts, the entries FirstChar, LastChar, Widths, and FontDescriptor shall either all be present or all be absent. Ordinarily, these dictionary keys may be absent; specifying them enables a standard font to be overridden; see 9.6.2.2, "Standard Type 1 fonts (standard 14 fonts) (PDF 1.0-1.7)".

For PDF 2.0. all of those fields are marked required, so there is **NEVER** a case where they can "all be absent". 

I would recommend we simply remove the paragraph.

**Comment by petervwyatt** [comment]
Proposed solution to delete para (as above)

**Comment by crlf0710** [comment]
I feel there's value in standardizing the values of these entries for 14 standard fonts (both for pdf writers and for interactive pdf processors which needs to actually ship compatible font data for rendering these), not necessarily in the standard text though.

**Comment by lrosenthol** [comment]
@crlf0710 what do you mean by "standardizing the values"?   What values?

**Comment by crlf0710** [comment]
@lrosenthol I meant providing reference values for `Widths`, `FontDescriptor.Ascent`, `FontDescriptor.Descent`, `FontDescriptor.FontBBox` and other formerly optional entries for the standard 14 fonts.

I'm hoping that different pdf writers can generate consistent standard font dictionary and font descriptors using this information, without any dependency on the current operating system or other environment factor.

**Comment by lrosenthol** [comment]
@crlf0710 Unfortunately, there is no such thing - which is part of why we've tried to abandon/deprecate the entire concept of the standard 14 for more than a decade.

**Comment by MPBailey** [comment]
There's a closely related issue in paras 2 and 3 of 9.6.2.2, almost immediately after the table.
I would recommend deleting the 2nd sentence of the 2nd para, and the whole of the 3rd para

In other words, delete "PDF processors supporting PDF 1.0 to PDF 1.7 files shall have these fonts, or their font metrics and suitable substitution fonts, available.
These fonts, or their font metrics and suitable substitution fonts, shall be available to the PDF processor."

**Comment by petervwyatt** [comment]
Wow. This is repeated and repeated and repeated.

I agree about removing all the excess words and saying this clearly just the once, but I think it might be better stated under 9.6.2.2 "Standard Type 1 fonts (standard 14 fonts) (PDF 1.0-1.7)" primarily because of the heading title, rather than as a misc floating para below Table 109 under subclause 9.6.2.1 General. We can also add some cross-referencing so everything points to 9.6.2.2

So to summarize an updated proposed solution:

* Table 109, **FontDescriptor**: delete 2nd para and replace with a "See also 9.6.2.2 Standard Type 1 fonts (standard 14 fonts) (PDF 1.0-1.7)." cross-reference forward pointer.

* Delete last (short) para from 9.6.2.2

Also changing label from "bug" to "documentation" as this was not incorrect, just stated repeatedly.

**Comment by MPBailey** [comment]
That sounds good, but I'd still also remove the 2nd sentence from the 2nd para of 9.6.2.2. It is not the job of this standard to tell people what they must do to support previous versions of PDF. At the very least the 'shall' in that sentence needs changing.

**Comment by petervwyatt** [comment]
PDF TWG agree that PDF 2.0 should not push requirements onto past versions of PDF specs. Changes are:

* reword to avoid normative language "... were required to have these fonts..."
* Make whole 2nd para as a note (informative).
* Delete para after Table 109 and replace with forward "See also reference to 9.6.2.2.

**Comment by petervwyatt** [comment]
See also #48
